_Sidebar.md

```diff
@@ -9,7 +9,7 @@
     * [[Voice artists | Instructions-for-voice-artists]]
     * [[Designers and artists|Contributing-to-Oppia's-design]]
   * Reporting a bug
-    * [[File an issue for bug|Github-Issue-Guide]]
```

Comment:
I remember now why I named it like this; For Lesson creation guide, it's named as `Create a Lesson!`, so I thought this would make more sense.

Reply:
I think that was intentionally phrased as an invitation, but here we aren't exactly inviting folks to file reports, just providing the guide for how to do it if they need to.

Reply:
Got it, Thanks!